Explorer/Assets/Scripts/Global/Dynamic/Bootstraper.cs

```diff
@@ -190,17 +183,20 @@ public async UniTask LoadStartingRealmAsync(DynamicWorldContainer dynamicWorldCo
         }
 
         public async UniTask UserInitializationAsync(DynamicWorldContainer dynamicWorldContainer,
-            GlobalWorld? globalWorld, Entity playerEntity, Animator splashScreenAnimation, GameObject splashRoot, CancellationToken ct)
+            GlobalWorld globalWorld, Entity playerEntity, ISplashScreen splashScreen, CancellationToken ct)
         {
-            if (showSplash)
-                await UniTask.WaitUntil(() => splashScreenAnimation.GetCurrentAnimatorStateInfo(0).normalizedTime > 1, cancellationToken: ct);
-
-            splashScreenAnimation.transform.SetSiblingIndex(1);
-
-            await dynamicWorldContainer.UserInAppInitializationFlow.ExecuteAsync(showAuthentication, showLoading, false,
-                globalWorld!.EcsWorld, playerEntity, ct);
+            splashScreen.ShowSplash();
```

Comment:
I think you can remove this invoke. The splash screen will be hidden by the loading screen

Reply:
Maybe it would work without it. I saved the previous flow just with `IScreenSplash`